lib/services/livesync/platform-livesync-service-base.ts

```diff
@@ -78,7 +79,7 @@ export abstract class PlatformLiveSyncServiceBase implements IPlatformLiveSyncSe
 			device.applicationManager.checkForApplicationUpdates().wait();
 
 			let appIdentifier = this.liveSyncData.appIdentifier;
-			if (!device.applicationManager.isApplicationInstalled(appIdentifier).wait()) {
+			if (!device.applicationManager.isApplicationInstalled(appIdentifier).wait() || !this.$platformService.isPlatformBuilt(deviceAppData.platform.toLowerCase()).wait()) {
```

Comment:
Why do we need to be sure that the platform is built? In case the application is installed on device, shouldn't we just livesync the files (persuming there are no changes requiring build).